src/helpers/axios.js

```diff
@@ -0,0 +1,3 @@
+import axios from 'axios';
```

Comment:
@dannylwe Instead of just export axios again make an axios instance so that it is used for all endpoints that need authentication. As for those that don't need authentication they can just import axios